src/gc.c

```diff
@@ -138,8 +146,12 @@ void *GC_thread(void *val)
 		if(now - lastResourceCheck >= RCinterval)
 		{
 			check_load();
-			check_space(FTLfiles.FTL_db);
-			check_space(FTLfiles.log);
+			int DBStorageUsage = check_space(FTLfiles.FTL_db, LastDBStorageUsage);
+			int LogStorageUsage = check_space(FTLfiles.log, LastLogStorageUsage);
+			
+			LastLogStorageUsage = LogStorageUsage;
+			LastDBStorageUsage = DBStorageUsage;
+
```

Comment:
```suggestion
			LastDBStorageUsage = check_space(FTLfiles.FTL_db, LastDBStorageUsage);
			LastLogStorageUsage = check_space(FTLfiles.log, LastLogStorageUsage);
```
No need for intermediate variables.

Reply:
Nice. I wasn't sure if this is supported and went for the safer way first.

Reply:
The way it works now is that it takes the value at the time of invoking the function (old value) and overwrites it only once the function returns (new value). Could be even fancier and pass the pointer to the memory location and do all the changing in the function, but no need for special fanciness. This works equally well.